packages/ffe-form-react/src/InputGroup.js

```diff
@@ -26,6 +27,8 @@ class InputGroup extends Component {
             ...rest
         } = this.props;
 
+        const id = childId ? childId : this.generatedId;
```

Comment:
You've already assigned `childId` to `this.generatedId` in the constructor if it is set, are we sure we want to allow consumers to change a field's ID after it has been mounted?

Reply:
No, that was not my intention. I just wanted the childId not to appear in the generated div element, if it is not present in line 18 it will be passed along with the ...rest. 

I see that I can achieve the same by adding an eslint-ignore, do you think that's a better solution, or do you have another suggestion?

Reply:
I'd just add an ignore in this case. I think it's even possible to configure eslint to not complain about unused variables when there's a `...rest` present but I can't remember how to do it.

Either way is fine with me, I just wanted to know if there was an actual usecase I wasn't seeing :) Techincally any ID we can be changed at any time so it's not really a big issue if this one can as well.

Reply:
Updated